README.md

```diff
@@ -33,3 +33,39 @@ make build
 ```
 make test
 ```
+
+## Local Devnet Deployment
+
+### Launch Anvil
+
+```
+make anvil-prague
+```
+
+### Deploy invoker
+```
+./bin/forge script Deploy --sig "deploy()" --rpc-url $RPC_URL --private-key $EXECUTOR_PRIVATE_KEY --broadcast
+```
+
+### Test invoker
```

Comment:
honestly i'm not sure these steps are ideal to have in the README, it's a little in the weeds i think better documented in code. e.g. implement a forge script that actually does all of these steps instead of copy/pasting params between scripts. 

like (roughly)
```
// script that does one arbitrary call via a BatchInvoker
function call(address recipient, uint256 value, bytes data) public {
   // construct batch execData
   bytes memory calls = packCalls("", recipient, value, data);
   uint256 batchNonce = BatchInvoker(invoker).nextNonce(vm.addr(apk));
   bytes memory execData = abi.encode(nonce, calls);

   // generic script for sign & execute for any invoker
   signAndExecute(execData);
}

// script that just sends eth
function sendEth(address recipient, uint256 value) {
   uint256 balanceBefore = recipient.balance;
   call(recipient, value, "");
   assertEq(recipient.balance, balanceBefore + value);
}
```

Reply:
Ok I'll take a pass and create a super simple make command for testing the invoker